src/sentry/api/helpers/group_index.py

```diff
@@ -467,7 +467,11 @@ def wrapper(*args, **kwargs):
                     window=window,
                 ):
                     return Response(
-                        {"detail": "You are attempting to use this endpoint too quickly."},
+                        {
+                            "detail": "You are attempting to use this endpoint too quickly. Limit is {}/{}s".format(
```

Comment:
nit: Maybe calculating the limit/window and using {rate}/s might be nicer but nbd (eg. 1/s instead of 1/1s)

Reply:
Yeah, I had a mini-debate about whether people would rather see things like 1.5/s or 3/2s. Considering our major use case is 1s, I think I'll implement your suggestion to support that better :-)